result/rsvp.yaml

```diff
@@ -0,0 +1,192 @@
+openapi: 3.1.0
+
+info:
+  title: RSVP-TE metric results model.
+  version: ^0.0.0
+
+components:
+  schemas:
+    Rsvp.Metrics.Request:
+      description: >-
+        The request to retrieve RSVP-TE per Router metrics/statistics.
+      type: object
+      properties:
+        router_names:
+          description: >-
+            The names of RSVP-TE Routers to return results for.
+            An empty list as input will return results for all RSVP-TE routers.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Rsvp/properties/name"
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the input list is empty then all columns will be returned except for
+            any result_groups.            
+          type: array
+          items:
+            type: string
+            enum:
```

Comment:
Given that this is such a long list, is there any particular order in which these are specified?  

If not, any particular order that we can use?  Alphabetical would be obvious, but perhaps there is some other type of order that we can follow which might make it easier for end user to find stats or know whether they are supported or not.

Reply:
The order is lsp status and then each type of protocol message's tx and rx starting with path and resv and then the tears. Then the errors and the the stats for the optional features.